docs/tctl/how-to-add-a-custom-search-attribute-to-a-cluster-using-tctl.md

```diff
@@ -11,6 +11,10 @@ tags:
   - tctl
 ---
 
+## NOTE
```

Comment:
no note headers - 

I generally try to avoid using note callouts for general documentation.
If we need to - let use the `:::` syntax